CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+### Version 9.3
```

Comment:
lol you'll have another rebase practice.. 9.2 hasn't been released, so please move this under 9.2